src/when.js

```diff
@@ -0,0 +1,32 @@
+var _curry3 = require('./internal/_curry3');
+
+
+/**
+ * Creates a function that will first pass its argument to the given predicate
+ * function. If the predicate is satisfied, the function will return the result
+ * of calling the `whenTrue` function with the same argument. If the predicate
+ * is not satisfied, the argument is returned as is.
+ *
+ * @func
+ * @memberOf R
+ * @category Logic
+ * @see R.ifElse, R.unless
+ * @sig (a -> Boolean) -> (a -> a) -> a -> a
+ * @param {Function} pred       A predicate function
+ * @param {Function} whenTrueFn A function to invoke when the `condition`
+ *                             evaluates to a truthy value.
```

Comment:
This is nitpicky, but "evaluates" should probably be indented to align with the "A".